src/app/common/components/errors/system-message/system-message.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-system-message',
+  templateUrl: './system-message.component.html',
+  styleUrls: ['./system-message.component.scss'],
+})
+export class SystemMessageComponent {
+
+    @Input() public type!: string;
+    @Input() public text!: string;
+    @Input() public description?: string = '';
+
+    public getBackgroundColor = (): string => this.type;
```

Comment:
This one can be removed. Instead you should improve type for `type` :) It should allow only strings for which you have CSS classes specified in CSS file. For now you have only `error` class, if you will remove default green background you will also have probably `success` class. That means that `type` property should have `'error' | 'success'` type :)